src/resources/i18n/translations/en.json

```diff
@@ -15,8 +15,8 @@
   "Account": "Account",
   "Active Downloads": "Active Downloads",
   "Actual amount will be higher due to network fees": "The actual amount also includes network fees",
-  "Add current item next in queue when playing a new item": "Add current item next in queue when playing a new episode",
-  "Add Custom RSS Feed": "Add Custom RSS Feed",
+  "Add current item next in queue when playing a new item": "Immediate playback",
+  "Add Custom RSS Feed": "Add custom RSS feed",
```

Comment:
The capital letters are intended for consistency:

![Screen Shot 2022-06-23 at 8 56 05 PM](https://user-images.githubusercontent.com/2262919/175446822-6dd71b1d-8f19-4a43-93e9-a64f7d60610e.png)

When text is used as a label for a button and/or the header of a page, I think capitalizing non-articles and non-prepositions should be fine. It's functioning as a title rather than ordinary text.

For example, Terms and Conditions:

![Screen Shot 2022-06-23 at 8 58 19 PM](https://user-images.githubusercontent.com/2262919/175447046-426e8181-9ded-4453-b982-357dddda1f2c.png)

It's a page header, and if we lowercased Conditions that would seem less natural to me.

And on the More screen, everything is uppercase...except Log out which is an oversight in terms of consistency by me:

![Screen Shot 2022-06-23 at 8 58 50 PM](https://user-images.githubusercontent.com/2262919/175447100-0783ab56-ba67-4663-8b28-d9b7dadc9778.png)

![Screen Shot 2022-06-23 at 8 59 30 PM](https://user-images.githubusercontent.com/2262919/175447161-0bd2661e-dffd-4092-80de-023b2025e504.png)

I get that maybe this capitalization convention doesn't translate to other languages very cleanly, but lowercasing our page titles doesn't seem the most natural way to go imo.

Reply:
I agree with buttons and actual titles, but not entries.
Moreover, "Mode: Podcasts" is not a sentence following the colon.
https://developers.google.com/style/capitalization#capitalization-in-titles-and-headings

Sentence case also avoids errors in other languages.

Reply:
@comradekingu would the list items' titles on the More screen be considered entries?

If that's what you recommend, part of the hurdle at the moment is we'd need to add new handling throughout the app to do something like "Add custom RSS feed" in the list item title, and "Add Custom RSS Feed" separately as the page header title. I'll need some time to sweep through the app and identify all the areas where the different casing should be used.

And our translation setup isn't optimal for concatenated things like "Mode: Podcasts". I suppose one thing we can do is avoid concatenation entirely, although some kind of string interpolation that is understandable through the JSON files might be better?

You can find the Mode text handling here. https://github.com/podverse/podverse-rn/blob/develop/src/screens/MoreScreen.tsx#L193

The idea for Mode btw is the app has different Modes, namely that you can put it in "Podcast" mode or "Video" mode. Someday we could add "Music" mode or "Audiobook" mode.

So anyway, it seems like maybe there are 2 larger issues to tackle here...1 would be sweeping through the whole app to identify and correct casing issues like you've raised. 2 would be to add some kind of string interpolation. If you can help create those issues it'd help me prioritize the work sooner.

Reply:
"More" has entries IMO.
Concatenation is best avoided, as then everything is in the right order, and a functional set of words can be picked.
Lets say "mode" is gendered, or a different word if it is audio.
This PR leaves it at (attempted) consistency wrt. capitalisation, and then I'll add issues for both concatenation and casing?

Reply:
Thanks for creating those'd like to work on standardizing casing across the web and mobile app...realistically might not be for a couple months though with all the stuff on my plate. I _might_ be able to get some extra help with this though soon. 

And sorry for the delay. I had a really busy month, and also was preparing for this podcast I was on yesterday 😅 If you ever want to check it out [here it is](https://www.podland.news/1538779/10961075-hint-it-s-really-simple-to-extend-standards).